src/Common/test/Common.TestResources/Steeltoe.Common.TestResources.csproj

```diff
@@ -16,7 +16,7 @@
     <PackageReference Include="xunit.abstractions" Version="2.0.3" />
   </ItemGroup>
 
-  <ItemGroup Condition="'$(TargetFramework)' == 'net6.0'">
+  <ItemGroup Condition="'$(TargetFramework)' == 'net8.0' Or '$(TargetFramework)' == 'net6.0'">
```

Comment:
I think this condition can go away now (pretty sure it original intent was `!netcoreapplication3.1`